sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/implementation/ManagementChannelTests.java

```diff
@@ -59,6 +59,7 @@
 import static org.junit.jupiter.api.Assertions.assertNull;
```

Comment:
Create an issue to test that new behaviour for transaction behaviour you added.

Reply:
https://app.zenhub.com/workspaces/azure-sdk-team-5bdca72c4b5806bc2bf0aab2/issues/azure/azure-sdk-for-java/11832